jaxopt/_src/quadratic_prog.py

```diff
@@ -177,7 +177,8 @@ def run(self,
           init_params: Optional[Tuple] = None,
           params_obj: Optional[ArrayPair] = None,
           params_eq: Optional[ArrayPair] = None,
-          params_ineq: Optional[ArrayPair] = None) -> base.OptStep:
+          params_ineq: Optional[ArrayPair] = None,
+          ) -> base.OptStep:
```

Comment:
Any good  reason for this change? If not, I would change back for consistency with the rest of the code base.